django/cantusdb_project/main_app/views/chant.py

```diff
@@ -750,17 +750,29 @@ def get_queryset(self) -> QuerySet:
         display_unpublished = self.request.user.is_authenticated
         # if the search is accessed by the global search bar
         if self.request.GET.get("search_bar"):
-            chant_set = Chant.objects.filter(source__published=True)
-            sequence_set = Sequence.objects.filter(source__published=True)
+            if display_unpublished:
+                chant_set = Chant.objects.all()
+                sequence_set = Sequence.objects.all()
+            else:
+                chant_set = Chant.objects.filter(source__published=True)
+                sequence_set = Sequence.objects.filter(source__published=True)
             if self.request.GET.get("search_bar").replace(" ", "").isalpha():
                 # if search bar is doing incipit search
```

Comment:
Is this comment correct?

Reply:
we just discussed this in person, but leaving a tiny summary in case people read this in the future: while it's true that it's not searching only the `incipit` field, it's searching for the search term at the very beginning of each of the text fields - from the user's point of view, they're searching for an incipit.